readme.md

```diff
@@ -86,17 +86,495 @@ Click the type names for complete docs.
+            Playground
+        </a>
+    </summary>
+
+    ```typescript
+    interface Config {
+        method: string;
+        async: boolean;
+        url: string;
+        query: string;
+    }
+
+    /** Get union of ObjectType' keys that extend TargetType */
+    type KeysOfType<ObjectType, TargetType> = {
+        [Key in keyof ObjectType]: ObjectType[Key] extends TargetType ? Key : never;
+    }[keyof ObjectType];
+
+
+    /** Omit those properties that extend ExcludeType */
+    type OmitTypeKeys<ObjectType, ExcludeType> = Pick<
+        ObjectType,
+        Exclude<keyof ObjectType, KeysOfType<ObjectType, ExcludeType>>
+    >;
+
+    type StringConfigKeys = KeysOfType<Config, string>; // 'method' | 'url' | 'query'
+
+    /**
+     * From keyof Config: `'method' | 'async' | 'url' | 'query'`
+     * exclude
+     * KeysOfType<Config, string>: `'method' | 'url' | 'async'`
+     * leaving `'async'`
+     */
+    type NonStringConfigKeys = Exclude<keyof Config, KeysOfType<Config, string>>;
+
+    /** Get only properties that do not extend type string from type Config */
+    type NonStringConfig = OmitTypeKeys<Config, string>;
+
+    const nonString: NonStringConfig = { async: true };
+    ```
+    </details>
+
 - [`Extract<T, U>`](https://github.com/Microsoft/TypeScript/blob/2961bc3fc0ea1117d4e53bc8e97fa76119bc33e3/src/lib/es5.d.ts#L1441-L1444) - Extract from `T` those types that are assignable to `U`.
+    <details>
+    <summary>
+        Example
+        <a href="https://typescript-play.js.org/?target=6#code/JYOwLgpgTgZghgYwgAgCoQiZBvAUM5EOAWwgC5kBnMKUAcwG59k47zCBXYgI2iYF9cuUJFiIUAQQAmHADZhkEAB6QQUymgxY8BAA5QA9jAiVKwAyArVaIRswDuBqAGsAokt3RgmJBRBdeKAEhAHoAKjDkADFDYmRnCABPDSNkAHluACsIBFREzwlkKAhiAwA3FDhZWXikjTAACzgFOGLkfRNMBQt0rJy8zwAhZDCQ3DB8ySkpYDBzIlkAaTqAHgzshDABiAkAGl6NrcnBgD5kAF5kdwRZDikIFeYExNT1nKOC3aek177N7cGuBOTFw4UiywgumQFlkiWQjQMlBQzxSMAO722hUazRYbRABgUHSR4GhWDeuWOIzGE08yAAIsAYDA1n8Pjt9uS2acLjhmABtZZw0DIaQzOYWKqCygsw6YjmsgEnAC6FE5mIFSSVuH4ILBaCgcBAlBgTji6EwyBEBhFMnkVNwMA4IE282QdEM9gAqroABSQTAUc0gfYcXRSZrsBlMlbSORgfZBk4ASgosbtOiKEDAHCgZNZADo4KZgHQQH6tCGwxGk8FcO6DF7fcxsIQSOwAEQAWQg7f2rHYAEYABzIfhfAgt-RGExmCwUdsASRAMFuPmgveQjhc7k8tDXFAADKPcDXcEA">
+            Playground
+        </a>
+    </summary>
+
+    ```typescript
+    interface Teen {
+        name: string;
+        age: number;
+    }
+
+    interface Adult extends Teen {
+        profession: string;
+        workExperience: number;
+    }
+
+    /** From keys of ObjecTypeA remove all keys that are present on ObjecTypeB */
+    type AdditionalKeys<ObjectTypeA, ObjectTypeB> = Exclude<
+        keyof ObjectTypeA,
+        keyof ObjectTypeB
+    >;
+
+    /** Keep only those keys of ObjectTypeA that are not present on ObjecTypeB */
+    type Diff<ObjectTypeA, ObjectTypeB> = {
+        [Key in AdditionalKeys<ObjectTypeA, ObjectTypeB>]: ObjectTypeA[Key]
+    };
```

Comment:
This is not directly related to `Exclude`. I would prefer the example to be simple without this.